### PR TITLE
Add switchboard and lexicon data preparation helper jobs

### PR DESCRIPTION
**Pull request by mmz33**

### CODE REVIEW

File: common/datasets/switchboard.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                phon=[tag_to_phon[tag]],
+                synt=[[]],
+                eval=[[]],
+            )
+        )
+
+    # create special lemmas
+    lex.add_lemma(
+        lexicon.Lemma(
+            orth=["[SENTENCE-END]"], synt=[["</s>"]], special="sentence-boundary"
```
Comment (michelwi) [inline common/datasets/switchboard.py:83]: any special reason this has no `eval=[[]]`?

Comment (mmz33) [reply common/datasets/switchboard.py:83]: hmm not sure why. I checked again the current lexicon we use and it is without `eval`.

Comment (mmz33) [reply common/datasets/switchboard.py:83]: the `sentence-end` lemma has the same `synt` token and there we have empty `eval` tag.

File: common/datasets/switchboard.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        sort_phonemes=True,
+        sort_lemmata=False,
+        compressed=True,
+    )
+    merge_lexicon_job.add_alias(os.path.join(alias_name, "merge_lexicon_job"))
+
+    return merge_lexicon_job.out_bliss_lexicon
+
+
+def _export_train_bliss_coprus(subdir_prefix=""):
```
Comment (JackTemaki) [inline common/datasets/switchboard.py:169]: typo `coprus`

File: common/datasets/switchboard.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+from i6_core.lexicon.modification import WriteLexiconJob, MergeLexiconJob
+from i6_core.lib import lexicon
+
+
+def get_speakers_list(subdir_prefix=""):
+    """
+    Returns speakers list
+
+    :param str subdir_prefix: alias name prefix
+    :rtype: tk.Path
```
Comment (JackTemaki) [inline common/datasets/switchboard.py:12]: if the `rtype` is only `Path`, please also specify with `:return:` in a line directly above what kind of file is returned.

**Review by JackTemaki — CHANGES_REQUESTED** [review]

**Review by JackTemaki — APPROVED** [review]

**Pull request merged**